python/paddle/fluid/tests/unittests/test_recurrent_op.py

```diff
@@ -124,6 +124,7 @@ def setup_program(self):
     def setUp(self):
         self.setup_program()
         self.data_field = {"x", "h_boot"}
```

Comment:
Their names are not good ... if you are a future developer who saw those lines, you will ask what's data_field, what's fetch_data_field? Why are they same in most cases? Also, if fetch_data_field is used for fetching, why do we still use data_field in "def forward(self)"....

I would suggest to change the name "data_field" to "feed_data_field"
"fetch_data_field" to "grad_data_field". In old test cases, you should write "self.grad_data_field = self.feed_data_field"

In the test case you added, you should comment "init doesn't have gradient"

Reply:
Done.